### ISSUE TITLE
M1 Mac Issues with ElasticSearch

### ISSUE THREAD

**Issue by zwing99** [localstack/localstack#3490]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

Using `localstack-full`, I am not able to use the elastic search service. The creation of indices never completes.

from logs I see this:
```python
Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/bootstrap.py", line 581, in run
    result = self.func(self.params)
  File "/opt/code/localstack/localstack/services/es/es_api.py", line 229, in do_start
    start_elasticsearch_instance(version=version)
  File "/opt/code/localstack/localstack/services/es/es_api.py", line 199, in start_elasticsearch_instance
    check_infra(apis=[], additional_checks=[es_starter.check_elasticsearch])
  File "/opt/code/localstack/localstack/services/plugins.py", line 105, in check_infra
    check_infra(retries - 1, expect_shutdown=expect_shutdown, apis=apis, additional_checks=additional_checks)
  File "/opt/code/localstack/localstack/services/plugins.py", line 105, in check_infra
    check_infra(retries - 1, expect_shutdown=expect_shutdown, apis=apis, additional_checks=additional_checks)
  File "/opt/code/localstack/localstack/services/plugins.py", line 105, in check_infra
    check_infra(retries - 1, expect_shutdown=expect_shutdown, apis=apis, additional_checks=additional_checks)
  [Previous line repeated 7 more times]
  File "/opt/code/localstack/localstack/services/plugins.py", line 103, in check_infra
    raise e
  File "/opt/code/localstack/localstack/services/plugins.py", line 99, in check_infra
    additional(expect_shutdown=expect_shutdown)
  File "/opt/code/localstack/localstack/services/es/es_starter.py", line 97, in check_elasticsearch
    assert isinstance(out, six.string_types)
AssertionError
```

## Expected behavior

I should be able to use on my m1 mac.

## Actual behavior

# Steps to reproduce

run local stack with es and create an index on an M1 mac.

**Comment by mgagliardo** [comment]
Hello @zwing99 I do not own an M1 Mac to test this out but will try to bring a new one on AWS. Can you please add `DEBUG=1` to your compose (if you are using it) and show me the full output? And if you can show me how are you running this as well it would be great.

Thanks!

**Comment by zwing99** [comment]
@mgagliardo I built a repo to reproduce the issue: https://github.com/zwing99/m1-localstack and added my intel mac vs m1 logs. Actually, it looks as though I cannot get either to work now. I have an older built image on a private repo that worked, so not entirely sure here.
https://github.com/zwing99/m1-localstack/blob/main/logs-m1.txt
https://github.com/zwing99/m1-localstack/blob/main/logs-intel.txt

**Comment by zwing99** [comment]
So @mgagliardo it looks like it might be in part tied to #3081 ... I went to version 11.4 in my docker-compose file and it now works just fine on Intel mac. but still does not work on m1 macs.d I pushed updates to the repo mentioned above.

**Comment by mgagliardo** [comment]
Hey @zwing99 sorry for the delay, I'll assign this myself and treat it as duplicate, please open a new issue if this keeps happening after #3081 is closed (I am actively working on a fix, will push a PR soon). Thanks!

**Comment by mgagliardo** [comment]
Hey @zwing99 , reporting here this issue should've been fixed via https://github.com/localstack/localstack/pull/3565 please give a try to the `latest` docker image (pull) and let me know in any case. Cheers!
